src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class RegistrationServiceImplTest {
+    public static final Long DEF_ID = 19L;
+    public static final String DEF_LOGIN = "asdf@gmail.com";
+    public static final String DEF_PASSWORD = "qweasd";
+    public static final Integer DEF_AGE = 18;
+    static User expected;
+    static RegistrationServiceImpl registrationService;
+    User actual;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        expected = new User(DEF_ID, DEF_LOGIN, DEF_PASSWORD, DEF_AGE);
+        User userExist = new User(1234L, "asdasd@gmail.com", "1234567", 65);
+        registrationService.register(userExist);
```

Comment:
let's move these two lines to the test where you need this user to be registered